Adding-dropdown-menu

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
```

Comment:
The code looks good, but I would expect to see this on the Search Results page, not on its own page.